tools/database/enchant_overrides.go

```diff
@@ -151,7 +151,7 @@ var EnchantOverrides = []*proto.UIEnchant{
 	{EffectId: 930, SpellId: 13947, Name: "Enchant Gloves - Riding Skill", Quality: proto.ItemQuality_ItemQualityUncommon, Stats: stats.Stats{}.ToFloatArray(), Type: proto.ItemType_ItemTypeHands},
 	{EffectId: 1887, SpellId: 20012, Name: "Enchant Gloves - Greater Agility", Quality: proto.ItemQuality_ItemQualityUncommon, Stats: stats.Stats{stats.Agility: 7}.ToFloatArray(), Type: proto.ItemType_ItemTypeHands},
 	// Drop in BRS
-	// {EffectId: 927, SpellId: 20013, Name: "Enchant Gloves - Greater Strength", Quality: proto.ItemQuality_ItemQualityCommon, Stats: stats.Stats{stats.Strength: 7}.ToFloatArray(), Type: proto.ItemType_ItemTypeHands},
+	{EffectId: 927, SpellId: 20013, Name: "Enchant Gloves - Greater Strength", Quality: proto.ItemQuality_ItemQualityCommon, Stats: stats.Stats{stats.Strength: 7}.ToFloatArray(), Type: proto.ItemType_ItemTypeHands},
```

Comment:
Will this enchant actually be available this phase? As far as I can tell it only drops from mobs in LBRS and from the phase announcement it sounds like LBRS won't be available
https://www.wowhead.com/classic/item=16244/formula-enchant-gloves-greater-strength

Reply:
I didn't know they are disabling LBRS, I'll comment it again then

Reply:
That's the assumption at least. The announcement mentioned visiting "Maraudon, Zul'Farrak, and Blackrock Depths" (in addition to the ST raid of course)